src/parcsr_ls/par_mgr_solve.c

```diff
@@ -787,7 +797,7 @@ hypre_MGRCycle( void               *mgr_vdata,
                hypre_ParCSRMatrixMatvecOutOfPlace(-1.0, A_array[level],
                                                   U_array[level], 1.0, F_array[level], Vtemp);
 
-               resnorm = hypre_ParVectorInnerProd(Vtemp, Vtemp);
+               resnorm = sqrt(hypre_ParVectorInnerProd(Vtemp, Vtemp));
```

Comment:
Was this a bug before?

Reply:
Yes from what I talked to Daniel, however the change didn't lead to any test failure. Maybe because MGR is only tested as a preconditioner? Perhaps we should look at testing the value of `resnorm` explicitly when using it as a solver, even if that's not common

Reply:
Yes, we probably should also test MGR as a standalone solver, if that's what you mean here.

Reply:
Yep, I asked the same q. to @oseikuffuor1 in the other PR.